maat-orchestration/src/test/java/uk/gov/justice/laa/crime/orchestration/service/orchestration/HardshipOrchestrationServiceTest.java

```diff
@@ -526,7 +569,10 @@ void givenExceptionThrownInValidationService_whenCreateIsInvoked_thenRollbackIsN
     @Test
     void givenExceptionThrownInValidationService_whenUpdateIsInvoked_thenRollbackIsNotInvoked() {
         WorkflowRequest workflowRequest = buildWorkflowRequestWithHardship(CourtType.MAGISTRATE);
-        doThrow(new CrimeValidationException(List.of())).when(validationService).isUserActionValid(any());
+        doThrow(new CrimeValidationException(List.of())).when(validationService).isUserActionValid(any(), any());
```

Comment:
Could still do with updating some of these calls throughout the test file.

Reply:
I think two of these instances have been written because `when(some_method_call)` expect a generic return type of `T` but are being set-up with method invocations which actually return `void`, resulting in a compile-time error.

The other two instances which return an actual type (calling `validationService.isUserActionValid`) have been updated.